tezos-release.nix

```diff
@@ -7,13 +7,10 @@ let
   version = replaceStrings ["refs/tags/"] [""] commonMeta.branchName;
   release-notes = writeTextDir "release-notes.md" ''
     This release contains assets based on the [${version} release](https://gitlab.com/tezos/tezos/tree/${version}).
-
     Binaries that target arm64 architecture have the `-arm64` suffix in their name.
     All the other binaries target x86_64.
-
     Release artifacts are signed with the following key: 0x7EAF9B150ACE940CF8C008A0BF847A85AC7BF43E.
     You can check it on http://keyserver.ubuntu.com/.
-
```

Comment:
Why the removed empty lines?

Reply:
To be honest something strange happens with `tezos-release.nix` every time I'm working on tezos-packaging
Some dev environment quirk

Reply:
this sounds odd, please revert these changes

Reply:
it's done